<?xml version="1.0" encoding="UTF-8"?><testrun name="MVCtest" project="Mjukvarutestning" tests="5" started="5" failures="1" errors="0" ignored="0">
  <testsuite name="tests.MVCtest" time="0.345">
    <testcase name="Test_systemExit" classname="tests.MVCtest" time="0.152">
      <failure>java.lang.AssertionError: System.exit has not been called.&#13;
	at org.junit.Assert.fail(Assert.java:88)&#13;
	at org.junit.contrib.java.lang.system.ExpectedSystemExit.handleMissingSystemExit(ExpectedSystemExit.java:149)&#13;
	at org.junit.contrib.java.lang.system.ExpectedSystemExit.checkSystemExit(ExpectedSystemExit.java:144)&#13;
	at org.junit.contrib.java.lang.system.ExpectedSystemExit.access$000(ExpectedSystemExit.java:88)&#13;
	at org.junit.contrib.java.lang.system.ExpectedSystemExit$1.evaluate(ExpectedSystemExit.java:133)&#13;
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)&#13;
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_MenuChoiceExceptionTest_NotInteger" classname="tests.MVCtest" time="0.001"/>
    <testcase name="test_ThrowAllDicesWithDifferentValues_Mock" classname="tests.MVCtest" time="0.19"/>
    <testcase name="test_MenuChoiceExceptionTest_NotInRange_TooHigh" classname="tests.MVCtest" time="0.001"/>
    <testcase name="test_MenuChoiceExceptionTest_NotInRange_TooLow" classname="tests.MVCtest" time="0.0"/>
  </testsuite>
</testrun>
